vendor/libraries/usabilitydynamics/lib-wp-list-table/lib/classes/class-wp-list-table.php

```diff
@@ -537,7 +537,7 @@ protected function parse_query( $query ){
                   $_query[ 'meta_query' ] = array();
                 }
 
-                if( is_array( $q['value'] ) && !in_array( $map['compare'], array( 'NOT IN', 'IN' ) ) ) {
+                if( is_array( $q['value'] ) && !in_array( $map['compare'], array( 'NOT IN', 'IN' ) ) && 'BETWEEN' != $map['compare'] ) {
                   $map['compare'] = 'IN';
```

Comment:
@mehulkaklotar This is a modification within a composer-installed library, are you aware that this'll require a modification in https://github.com/UsabilityDynamics/lib-wp-list-table/blob/master/lib/classes/class-wp-list-table.php#L540 as well?